tests/cover/test_direct_strategies.py

```diff
@@ -280,3 +280,21 @@ def test_none_lists_respect_max_size(ls):
 )
 def test_none_lists_respect_max_and_min_size(ls):
     assert 1 <= len(ls) <= 5
+
+
+@given(ds.iterables(ds.integers(), max_size=5, min_size=1))
+def test_iterables_are_exhaustible(it):
+    for _ in it:
+        pass
+    with pytest.raises(StopIteration):
+        next(it)
+
+
+@given(ds.iterables(ds.integers()))
+def test_iterables_repr_is_useful(it):
+    # fairly hard-coded but useful; also ensures _values are inexhaustible
+    assert repr(it) == 'iter({!r})'.format(it._values)
```

Comment:
Tests for pretty repr-printing usually live in `test_custom_reprs.py`.

Not super happy about the hard-coding here, but also struggling to see a good way out right now.